src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -11,10 +11,10 @@
 import edu.kis.powp.jobs2d.command.gui.CommandManagerWindow;
 import edu.kis.powp.jobs2d.command.gui.CommandManagerWindowCommandChangeObserver;
 import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
-import edu.kis.powp.jobs2d.events.SelectLoadSecretCommandOptionListener;
-import edu.kis.powp.jobs2d.events.SelectRunCurrentCommandOptionListener;
-import edu.kis.powp.jobs2d.events.SelectTestFigure2OptionListener;
-import edu.kis.powp.jobs2d.events.SelectTestFigureOptionListener;
+import edu.kis.powp.jobs2d.events.*;
+// import edu.kis.powp.jobs2d.events.SelectRunCurrentCommandOptionListener;
+// import edu.kis.powp.jobs2d.events.SelectTestFigure2OptionListener;
+// import edu.kis.powp.jobs2d.events.SelectTestFigureOptionListener;
```

Comment:
In revision repo it is safe to delete code instead of using a comment.

Reply:
Ahh, it was done just temporarily while merging changes (another group imported whole `edu.kis.powp.jobs2d.events.*`) – it seems that we forgot to remove it later on. Sorry.